.env_SAMPLE

```diff
@@ -0,0 +1,10 @@
+//export WORDPRESS=http://wordpress.domain
```

Comment:
This could possibly use a code comment at the top that says what it's used for. Like `// See INSTALL.md#4-project-configuration for details.`